presto-main/src/main/java/com/facebook/presto/operator/scalar/ReflectionParametricScalar.java

```diff
@@ -15,134 +15,114 @@
 
```

Comment:
The rename of `ScalarHeaderAndImplementations` should be done in the previous commit, since that class was introduced in this PR